jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java

```diff
@@ -197,6 +195,7 @@ public static WebDriver getChromeDriver(ChromeOptions chromeOptions) {
    * @return the chrome driver
    */
   public static WebDriver getChromeDriver(ChromeOptions chromeOptions, String size) {
+    WebDriverManager.chromedriver().setup();
```

Comment:
Would it be possible, reasonable to add this as something like a lazy initializer.
This should only need to run once.

Reply:
I'll do a quick double check.  The WDM should look at the local cache first unless we set `wdm.forceDownload=true`.